tasks/create.yaml

```
@@ -47,6 +51,7 @@ tasks:
     actions:
       - description: "Build the UDS Core Pepr Module"
         cmd: |
+          CUSTOM_PEPR_IMAGE=$( [ "${FLAVOR}" = "registry1" ] && echo "--custom-image ${REGISTRY1_PEPR_IMAGE}" ) || CUSTOM_PEPR_IMAGE=""
```

```suggestion
          CUSTOM_PEPR_IMAGE=$( [ "${FLAVOR}" = "registry1" ] && echo "--custom-image ${REGISTRY1_PEPR_IMAGE}" )
```
You can drop the `||` here since the evaluation of the first will result in nothing if flavor is not registry1.

---

i get this error when i remove the `||`

```bash
 NOTE  Saving log file to /tmp/runner-2024-03-13-09-31-19-1293762708.log
  •  Running "Build the UDS Core Pepr Module"
     ERROR:  Failed to run action: command "Build the UDS Core Pepr Module" timed out after 0 seconds
```

---

Interesting...it worked locally for me with the suggested change 🤔

---

i'm currently running latest uds from main, so tried `0.9.2` and got same thing. Maybe just an issue with my bash then?

---

I'm not sure if it was my uds version or something else but seeing the same on uds 0.9.2 🙃. Resolving!